apiserver/internal/handlers/resource/interface.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2025 Canonical Ltd.
```

Comment:
Just dump this file in head of resource.go. This is antipattern that we want to avoid.

Reply:
@SimonRichardson @Aflynn50, not sure I agree with this. Let's discuss at standup tomorrow (Tuesday).